dll/win32/comdlg32/lang/cdlg_Ro.rc

```diff
@@ -352,174 +352,174 @@ STYLE DS_SHELLFONT | DS_MODALFRAME | WS_POPUP | WS_VISIBLE | WS_CAPTION | WS_SYS
-CAPTION "Deschide"
+CAPTION "Deschidere"
 FONT 8, "MS Shell Dlg"
 {
-    LTEXT "&Caută în:", IDC_LOOKINSTATIC, 28, 6, 43, 8, SS_NOTIFY
+    LTEXT "&Privire în:", IDC_LOOKINSTATIC, 28, 6, 43, 8, SS_NOTIFY
     COMBOBOX IDC_LOOKIN, 64, 3, 150, 100, CBS_DROPDOWNLIST | CBS_OWNERDRAWFIXED | CBS_HASSTRINGS | WS_VSCROLL | WS_TABSTOP
 
     LTEXT "" , IDC_TOOLBARSTATIC, 209, 2, 102, 17, NOT WS_GROUP | NOT WS_VISIBLE
     CONTROL "", IDC_TOOLBARPLACES, "ToolbarWindow32", 0x800 | CCS_NORESIZE | CCS_TOP | CCS_NOPARENTALIGN | CCS_NODIVIDER, 4, 20, 56, 192
     LISTBOX IDC_SHELLSTATIC, 64, 20, 300, 155, LBS_SORT | LBS_NOINTEGRALHEIGHT | LBS_MULTICOLUMN | WS_HSCROLL | NOT WS_VISIBLE
 
-    LTEXT "N&ume fișier:", IDC_FILENAMESTATIC, 65, 182, 56, 16, SS_NOTIFY
+    LTEXT "&Nume fişier:", IDC_FILENAMESTATIC, 65, 182, 56, 16, SS_NOTIFY
     EDITTEXT IDC_FILENAME, 123, 180, 180, 12, ES_AUTOHSCROLL
     CONTROL "", cmb13, "ComboBoxEx32", CBS_DROPDOWN | CBS_AUTOHSCROLL | WS_VSCROLL | WS_TABSTOP, 123, 180, 180, 150
 
-    LTEXT "Tip de &fișier:", IDC_FILETYPESTATIC, 65, 200, 56, 16, SS_NOTIFY
+    LTEXT "&Fişiere de tipul::", IDC_FILETYPESTATIC, 65, 200, 56, 16, SS_NOTIFY
     COMBOBOX IDC_FILETYPE, 123, 198, 180, 53, CBS_DROPDOWNLIST | WS_VSCROLL | WS_TABSTOP
 
-    CONTROL "Nem&odificabil", IDC_OPENREADONLY, "Button", BS_AUTOCHECKBOX | WS_TABSTOP, 123, 218, 150, 10
+    CONTROL "Desc&hidere doar în citire", IDC_OPENREADONLY, "Button", BS_AUTOCHECKBOX | WS_TABSTOP, 123, 218, 150, 10
 
-    DEFPUSHBUTTON "&Deschide", IDOK, 310, 180, 54, 14
-    PUSHBUTTON "A&nulează", IDCANCEL, 310, 198, 54, 14
-    PUSHBUTTON "&Manual…", pshHelp, 310, 215, 54, 14
+    DEFPUSHBUTTON "&Deschidere", IDOK, 310, 180, 54, 14
+    PUSHBUTTON "Revocare", IDCANCEL, 310, 198, 54, 14
+    PUSHBUTTON "Aj&utor", pshHelp, 310, 215, 54, 14
 }
 
 NEWFILEOPENV3ORD DIALOG 0, 0, 440, 300
 STYLE DS_SHELLFONT | DS_MODALFRAME | DS_CONTEXTHELP | WS_VISIBLE | WS_POPUP | WS_CAPTION | WS_SYSMENU | WS_CLIPCHILDREN |
       WS_THICKFRAME
-CAPTION "Deschide"
+CAPTION "Deschidere"
 FONT 8, "MS Shell Dlg"
 {
-    LTEXT           "N&ume fișier:", IDC_FILENAMESTATIC, 160, 240, 60, 9, SS_RIGHT | WS_CLIPSIBLINGS
+    LTEXT           "&Nume fişier:", IDC_FILENAMESTATIC, 160, 240, 60, 9, SS_RIGHT | WS_CLIPSIBLINGS
     EDITTEXT        IDC_FILENAME, 226, 240, 100, 12,  WS_CHILD | WS_VISIBLE | WS_TABSTOP | WS_CLIPSIBLINGS | ES_AUTOHSCROLL
 
-    LTEXT           "Tip de &fișier:", IDC_FILETYPESTATIC, 160, 256, 60, 9, SS_RIGHT | WS_CLIPSIBLINGS
+    LTEXT           "&Fişiere de tipul:", IDC_FILETYPESTATIC, 160, 256, 60, 9, SS_RIGHT | WS_CLIPSIBLINGS
     COMBOBOX        IDC_FILETYPE, 226, 256, 100, 60,  WS_CHILD | WS_VISIBLE | WS_TABSTOP | WS_VSCROLL |
                     WS_CLIPSIBLINGS | CBS_HASSTRINGS | CBS_DROPDOWNLIST
 
-    DEFPUSHBUTTON   "&Deschide",  IDOK,     350, 240, 40, 14, WS_GROUP | WS_CLIPSIBLINGS
+    DEFPUSHBUTTON   "&Deschideree",  IDOK,     350, 240, 40, 14, WS_GROUP | WS_CLIPSIBLINGS
```

Comment:
```suggestion
    DEFPUSHBUTTON   "&Deschidere",  IDOK,     350, 240, 40, 14, WS_GROUP | WS_CLIPSIBLINGS
```
That is definitely a bug/typo.
Furthermore the question: Does the new "&Deschidere" fit the buttons 40 width?

Reply:
Yes, it should. Here is the proof.

![2024-04-30 00_21_15-Dialog - 1547](https://github.com/reactos/reactos/assets/113553683/caeaf182-f70b-4999-81bd-4a5377ee9409)